pkg/cloudprovider/providers/openstack/openstack_loadbalancer.go

```diff
@@ -1033,66 +1050,32 @@ func (lbaas *LbaasV2) ensureOctaviaHealthMonitor(lbID string, pool *v2pools.Pool
 	return nil
 }
 
-// Make sure the pool is created for the Service, nodes are added as pool members.
-func (lbaas *LbaasV2) ensureOctaviaPool(lbID string, listener *listeners.Listener, service *corev1.Service, port corev1.ServicePort, nodes []*corev1.Node, svcConf *serviceConfig) (*v2pools.Pool, error) {
-	var members []v2pools.BatchUpdateMemberOpts
-	newMembers := sets.NewString()
-
-	for _, node := range nodes {
-		addr, err := nodeAddressForLB(node)
-		if err != nil {
-			if err == ErrNotFound {
-				// Node failure, do not create member
-				klog.Warningf("Failed to get the address of node %s for creating member: %v", node.Name, err)
-				continue
-			} else {
-				return nil, fmt.Errorf("error getting address of node %s: %v", node.Name, err)
-			}
-		}
-
-		member := v2pools.BatchUpdateMemberOpts{
-			Address:      addr,
-			ProtocolPort: int(port.NodePort),
-			Name:         &node.Name,
-			SubnetID:     &svcConf.lbMemberSubnetID,
-		}
-		members = append(members, member)
-		newMembers.Insert(fmt.Sprintf("%s-%d", addr, member.ProtocolPort))
+func (lbaas *LbaasV2) buildMonitorCreateOpts(lbName string, port corev1.ServicePort) v2monitors.CreateOpts {
```

Comment:
Please add docstring for this method especially for why `PoolID` is not included in the return struct, to avoid confusion in the future.